templates/gsoc.html

```diff
@@ -0,0 +1,32 @@
+<html>
+
+    <head>
+        <title> GSoC 2017 Data </title>
+    </head>
+
+    <body>
+
+        {% if error %}
+            <h3> {{error}} </h3>
```

Comment:
This is not actually error log. It appears in case when there are no data for a org. I am caching "no data present" also because it was stated in issue that we need to cache the fact that there is not data present for a org. Probably i should rename it? :).

Reply:
> It appears in case when there are no data for a org.

Reply:
I have fixed this part. If related yaml file is not found then we show error.

>That is an error. It means the GSOC app needs to be disabled.

I do not deleted existing yaml files on new query. Should i delete existing yaml files so that if query is made for another year(for which there is no data) so it does not render data from existing files. If there is no data no file gets created.

Reply:
Remove this!  Do not render errors here.  This *app* should not exist if there was no gsoc data.  it isnt a gsoc org. no app. no yaml. no page.  using `logging` to record the gsoc app was disabled. no other visible trace.